web-wasi/Toolchain.cmake

```diff
@@ -1,7 +1,7 @@
 set(WASI_SDK_PREFIX $ENV{WASI_SDK_PATH})
 include($ENV{WASI_SDK_PATH}/share/cmake/wasi-sdk.cmake)
 
-set(CMAKE_FIND_ROOT_PATH $ENV{CROSS_ROOT})
+list(APPEND CMAKE_FIND_ROOT_PATH ${CMAKE_PREFIX_PATH} $ENV{CROSS_ROOT})
```

Comment:
Did you intent to change those as well? Or did they just get caught in a `sed` expression?

Might make sense, but those don't fit in the same pattern as the other `Toolchain.cmake`.

Reply:
Yeah I changed it on purpose:

1. The toolchain should never set CMAKE_FIND_ROOT_PATH, because it prevents the user from defining it (and it makes it confusing because -DCMAKE_FIND_ROOT_PATH is ignored, which happened to me). Instead it should append.
2. I really don't see a reason not to add CMAKE_PREFIX_PATH there. If it is unset, then nothing changes. And if it is set, it means that somehow there is an expectation that it will not be ignored :sweat_smile: